class UsersDALError(Exception):
    pass


class AuthDALError(Exception):
    pass
